pg_search/src/postgres/customscan/pdbscan/exec_methods/top_n.rs

```diff
@@ -65,10 +66,16 @@ impl TopNScanExecState {
             limit,
             sort_direction,
             need_scores,
+            is_partial_sort: false,
             ..Default::default()
         }
     }
 
+    pub fn set_partial_sort(&mut self, is_partial_sort: bool) -> &mut Self {
+        self.is_partial_sort = is_partial_sort;
+        self
+    }
```

Comment:
We haven't been doing this so far, but I'd love to begin adding comments on new builder fields that we add, to explain how they drive execution.

Reply:
Good idea. For now, this is completely removed.